### PR TITLE
[RFR] Auth0

### PR DESCRIPTION
**Pull request by SeppeDev**

### CODE REVIEW

**Comment by BennoDev** [comment]
Is there a way to pull out the Auth0 configuration part into a separate function? We want to avoid having the Startup and Configure functions grow out of hand.

**Comment by SeppeDev** [comment]
Login works, but now going to get the Auth0 logic out of the Startup and Configure, for readability, as Benno suggested above.

**Comment by SeppeDev** [comment]
As the ExerciseController and Views where not merged jet, I'll create a separate PR to add restrictions in there.

File: zwekness_api/zwekness_admin/Controllers/AccountController.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    {
+        private readonly Auth0Settings _auth0Settings;
+
+        public AccountController(IOptions<Auth0Settings> auth0Settings)
+        {
+            _auth0Settings = auth0Settings.Value;
+        }
+
+        [HttpGet]
+        public IActionResult Login(string returnUrl = "/")
```
Comment (BennoDev) [inline zwekness_api/zwekness_admin/Controllers/AccountController.cs]: You can annotate parameters from a get query string with [FromQuery]. Like this:
``` csharp
public IActionResult Login([FromQuery] string returnUrl = "/")
```

Comment (BennoDev) [inline zwekness_api/zwekness_admin/Controllers/AccountController.cs]: That is if they are in the URL string, and not in the post body.

Comment (SeppeDev) [inline zwekness_api/zwekness_admin/Controllers/AccountController.cs]: Currently there is none at all, so it just uses the default. Would it be better to put it in the URL string, and use this FromQuery, or is the post body the better option. Then I can implement it, or leave it out, and know which to use, when I change that returnUrl.

Comment (BennoDev) [inline zwekness_api/zwekness_admin/Controllers/AccountController.cs]: redirects belong in the url string typically

**Pull request merged**